scripts/environment.sh

```diff
@@ -1,21 +1,25 @@
+# Set the base paths to working directories.
+# Variables defined here use the prefix ns_
+set_working_paths() {
+    if [ -z "$ns_prefix" ]; then
+	echo "error: empty ns_prefix"
```

Comment:
You can use the `err` helper function here to make for consistent error printing.

Reply:
This is more of an internal error at this point: it 'shouldn't happen' but we should definitely stop if it does. Using `err` here would require sourcing `util.sh` from within the generated config/env scripts, which I'd rather avoid.